src/app/models/user.go

```diff
@@ -0,0 +1,140 @@
+package models
+
+import (
+	"crypto/rand"
+	"errors"
+	"log"
+	"math/big"
+	"time"
+
+	r "github.com/dancannon/gorethink"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// User holds attribute for an Ease user.
+type User struct {
+	ID           string    `gorethink:"id,omitempty" json:"id"`
+	Username     string    `gorethink:"username" json:"username"`
+	PasswordHash string    `gorethink:"password_hash" json:"-"`
+	APIToken     string    `gorethink:"api_token" json:"api_token"`
+	LoginToken   string    `gorethink:"login_token" json:"login_token"`
```

Comment:
I'm just curious - why?  Especially since we don't have a reset token feature, so this will always be equal to the CreatedAt time.